api/lib/mno_enterprise/concerns/controllers/jpi/v1/impac/dashboards_controller.rb

```diff
@@ -95,6 +95,7 @@ def dashboard_params
         whitelisted[:settings] = params[:dashboard][:metadata] || {}
       end
       .except(:metadata)
+      .merge(owner_type: "User", owner_id: current_user.id)
```

Comment:
we could change the api to accept an owner_type externally, that could be User or Organization